### PR TITLE
fix the documentation of normcase

### PR DESCRIPTION
**Pull request by kianasun**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/os.path.rst [hunk 1]
```diff
@@ -306,11 +306,12 @@ the :mod:`glob` module.)
 
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
-   Raise a TypeError if the type of *path* is not ``str`` or ``bytes`` (directly
-   or indirectly through the :class:`os.PathLike` interface).
+   Normalize the case of a pathname.  On Windows and Macintosh, convert all
+   characters in the pathname to lowercase, and on Windows also convert forward
```
Comment (Mariatta) [inline Doc/library/os.path.rst]: This seems incorrect. On my Mac it does not lowercase.

**Review by Mariatta — CHANGES_REQUESTED** [review]

File: Doc/library/os.path.rst (hunk 1 above)
Comment (merwok) [reply Doc/library/os.path.rst]: Python used to work on Mac 9 which was very different from windows or unix, before they made the unix-y Mac OS from a BSD.  In this diff, using «Macintosh» is a bit weird to me.

Comment (kianasun) [reply Doc/library/os.path.rst]: @Mariatta  @merwok I followed the terminology you used in [macpath.py](https://github.com/python/cpython/blob/master/Lib/macpath.py#L1).

Comment (Mariatta) [reply Doc/library/os.path.rst]: macpath has also been deprecated and no longer maintained, so I would not copy anything from macpath :)

Everywhere else we've been using `Mac OS X`.

Comment (merwok) [reply Doc/library/os.path.rst]: Exactly, macpath supports old-style Mac 9, not the current one.  Don’t use that module as a reference.

Comment (kianasun) [reply Doc/library/os.path.rst]: OK, I will change it.

**Comment by skx19952** [comment]
Hi, I have made changes. :)

File: Doc/library/os.path.rst [hunk 2]
```diff
@@ -306,10 +306,10 @@ the :mod:`glob` module.)
...
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
-   Raise a TypeError if the type of *path* is not ``str`` or ``bytes`` (directly
+   Normalize the case of a pathname.  On Windows, converts all characters in the
+   pathname to lowercase, and also converts forward slashes to backward slashes.
+   On other operating systems, return the path unchanged.
+   Raise a :exc:`TypeError` if the type of *path* is neither ``str`` nor ``bytes`` (directly
```
Comment (merwok) [inline Doc/library/os.path.rst]: In my opinion changing to neither/nor here doesn’t improve the text.

Comment (kianasun) [reply Doc/library/os.path.rst]: @merwok   I think neither/nor are more explicit and formal, see https://english.stackexchange.com/a/10061/44497

Comment (merwok) [reply Doc/library/os.path.rst]: Yes, but here I think the formalness makes the text heavier, less nice.

**Review by merwok — CHANGES_REQUESTED** [review]
revert `nor` + add `bpo-` reference in title

**Comment by csabella** [comment]
@kianasun Please resolve the file conflict and address the last review by @merwok.  Thanks!

**Comment by brettcannon** [comment]
@kianasun you also still need to sign the CLA.

**Comment by kianasun** [comment]
@brettcannon I have signed the CLA.

**Comment by brettcannon** [comment]
@kianasun also note that your PR is currently listed as changing 2901 files. Please check you aren't accidentally merging in another maintenance branch into master.

**Comment by merwok** [comment]
It looks like you created a virtualenv and added it to one of your commits.

Tips: `git add -p`, `git status` and `git diff --staged` help avoid these issues.

`git add .` is a risky command.

**Comment by kianasun** [comment]
Hi @brettcannon and @merwok , thanks for the tip! I made some changes. :)

**Review by merwok — APPROVED** [review]

**Comment by brettcannon** [comment]
While we wait for the CLA signing to register, @kianasun , do note there is a merge conflict to fix.

**Comment by kianasun** [comment]
@brettcannon Ok. I resolved the conflict.

**Comment by csabella** [comment]
@kianasun, the bot still doesn't seem to be finding your signed CLA.  Please make sure that your github ID matches the CLA.  Thanks!

**Comment by kianasun** [comment]
@csabella Hi, I have signed a new CLA. Thanks!

File: Doc/library/os.path.rst [hunk 3]
```diff
@@ -324,9 +324,9 @@ the :mod:`glob` module.)
 
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
+   Normalize the case of a pathname.  On Windows, converts all characters in the
+   pathname to lowercase, and also converts forward slashes to backward slashes.
+   On other operating systems, return the path unchanged.
```
Comment (vadmium) [inline Doc/library/os.path.rst:329]: IMO the middle sentence either needs a subject (_this_ converts . . .), like the old sentence had, or it needs changing to match the imperative (?) mood of the first and last sentences (_convert_ all characters . . .).

Comment (merwok) [reply Doc/library/os.path.rst:329]: +1 to imperative

Comment (kianasun) [reply Doc/library/os.path.rst:329]: Right, I am going to change "converts" to "convert".

**Comment by matrixise** [comment]
Hi @kianasun 

I have checked twice but I don't find your CLA, I have asked to @ewdurbin if he can check as well.

**Comment by ewdurbin** [comment]
No CLA has been submitted matching the users name or bpo user name.

**Comment by kianasun** [comment]
@matrixise Hi, I am so sorry. I signed it but I might forget to confirm it.. Can you please check it again? Sorry..

**Pull request merged**